testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -33,9 +34,22 @@
 from MDAnalysis.lib import mdamath
 from MDAnalysisTests import parser_not_found
 
-class _TestDistanceArray(TestCase):
-    # override backend in test classes
-    backend = None
+@pytest.fixture()
+def ref_system():
+    box = np.array([1., 1., 2.], dtype=np.float32)
+    points = np.array(
+        [
+            [0, 0, 0], [1, 1, 2], [1, 0, 2],  # identical under PBC
+            [0.5, 0.5, 1.5],
+        ], dtype=np.float32)
+    ref = points[0:1]
+    conf = points[1:]
+
+    return box, points, ref, conf
+    
+
+@pytest.mark.parametrize('backend', ['serial', 'openmp'])
```

Comment:
This is also really neat when  you have a `TestAFooBAR` that shouldn't run `test_a`.  So far we had do redefine the method with a pass statement or known failure.